### ISSUE TITLE
Unify pedestal button variant instances

### ISSUE THREAD

**Issue by Luke18033** [BEEmod/BEE2-items#4272]
Currently we have 3 different pedestal button items (original, adjustable, protruding) that use separate instances despite being functionally identical. These should be changed to all use the same instances, just positioned differently and with different templates depending on the item.

**Comment by TeamSpen210** [comment]
Definitely. When originally made, the compiler didn't have the math code to do the rotations, but now it does so we can unify.
